cypress.json

```diff
@@ -0,0 +1,6 @@
+{
+    "baseUrl": "http://localhost:19006",
+    "fixturesFolder": false,
+    "supportFile": false,
+    "pluginsFile": false
```

Comment:
this ensures that every time cypress is run it does not generate its boilerplate folders that we currently do not need.